Orange/widgets/data/owdatasampler.py

```diff
@@ -46,6 +46,9 @@ class OWDataSampler(OWWidget):
     number_of_folds = Setting(10)
     selectedFold = Setting(1)
 
+    class Warning(OWWidget.Warning):
+        could_not_stratify = Msg("Could not stratify\n{}")
```

Comment:
Could you turn this message into a sentence, something like "Stratification failed"?